CONTRIBUTING.adoc

```diff
@@ -1,9 +1,21 @@
+:icla: link:INDIVIDUAL_CONTRIBUTOR_LICENSE.adoc[Individual Contributor License Agreement]
```

Comment:
You should add `(ICL)` to the end of the text that renders for this link since you're using it as an acronym later in this contributing guide.

Reply:
The other option would be to just change the one ICL reference I see.

Reply:
Actually, `icla` in this case is a variable in the Asciidoc. When you view the rendered text, it's just a link to the agreement.

Reply:
but, you're right, I will add the ICL and use that in the reference in the rest of the document.